config/locales/en/course/condition/achievements.yml

```diff
@@ -1,6 +1,10 @@
 en:
   course:
     condition:
+      errors:
+        duplicate: 'An achievement cannot have duplicate conditions.'
+        itself: 'An achievement cannot have itself as a condition.'
+        none_selected: 'Please select a condition.'
```

Comment:
I think this is generated by ActiveRecord for the case where the field is required. Why is it necessary for you to manually trigger this error?